File: Python/Transacciones/Empresa.py
import math

class Empresa(object):
    def __init__(self, razon_social, ruc):
        self.razon_social = razon_social
        self.ruc = ruc
        self.pasos = 0

    def registrar_pasos(self, pasos):
        self.pasos = pasos

    def calcular_pago(self):
        if self.pasos <= 1000:
            return 500
        pasos_adicionales = math.ceil((self.pasos - 1000) / 100.00)
        return 500 + pasos_adicionales * 20

File: Javascript/Transacciones/Empresa.js
var Empresa = function(razon_social, ruc) {
    this.razon_social = razon_social;
    this.ruc = ruc;
    this.pasos = 0;

    this.registrar_pasos = function(pasos) {
        this.pasos = pasos;
    }

    this.calcular_pago = function() {
        if (this.pasos <= 1000) {
            return 500;
        }
        var pasos_adicionales = Math.ceil((this.pasos - 1000) / 100);
        return 500 + pasos_adicionales * 20;
    }
}
